app/src/main/java/org/wikipedia/page/references/PageReferences.java

```diff
@@ -0,0 +1,54 @@
+package org.wikipedia.page.references;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.google.gson.annotations.SerializedName;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+@SuppressWarnings("unused,NullableProblems")
+public class PageReferences {
+    @Nullable @SerializedName("selectedIndex") private int selectedIndex;
```

Comment:
The `@SerializedName` annotation is not necessary when the name of the field is the same.